updatecli/policies/open-release-pr/updatecli.d/open-release-pr.yaml

```diff
@@ -0,0 +1,164 @@
+name: Open a PR that prepares the policy for release
+pipelineid: "{{ .pipelineid }}"
+
+sources:
+  newestGitHubDraftReleaseVersion:
+    name: Extract version from newest GitHub draft release from release drafter
+    kind: githubrelease
+    spec:
+      owner: "{{ .github.owner }}"
+      repository: "{{ .github.repo }}"
+      token: "{{ requiredEnv .github.token }}"
+      key: "title"
+      typefilter:
+        draft: true
+      versionfilter:
+        kind: latest
+    transformers:
+      - trimprefix: "v"
+
+  newestReleaseTagVersion:
+    name: Extract newest tag from newest GitHub release
+    # used for displaying a gh compare URL
+    # disablesourceinput: true # we continue on fail (we may not have a tag)
+    kind: githubrelease
+    spec:
+      owner: "{{ .github.owner }}"
+      repository: "{{ .github.repo }}"
+      token: "{{ requiredEnv .github.token }}"
+      key: "title"
+      versionfilter:
+        kind: latest
+
+  cargoPackageName:
+    name: Extract cargo package name from Cargo.toml
+    kind: toml
+    scmid: "policy-repo"
+    dependson:
+      - condition#checkCargoTomlExists
+    spec:
+      file: "Cargo.toml"
+      key: "package.name"
+
+conditions:
+  checkCargoTomlExists:
```

Comment:
as it stands right now, these conditions fail hard when the Cargo.toml or Cargo.lock files are not present, and the policy fails on policies that are not Rust based:

```
DEBUG: Error while reading file(s): "✗ The specified file \"Cargo.lock\" does not exist. If you want to create it, you must set the attribute 'spec.forcecreate' to 'true'"

```

Reply:
Do you have an example of a policy where those files do not exist?

Reply:
responded out of band, but https://github.com/kubewarden/hostpaths-psp-policy.